bugsnag-plugin-android-exitinfo/src/test/java/com/bugsnag/android/ExitInfoCallbackTest.kt

```diff
@@ -0,0 +1,69 @@
+package com.bugsnag.android
+
+import android.app.ActivityManager
+import android.app.ApplicationExitInfo
+import android.content.Context
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.ArgumentMatchers.any
+import org.mockito.ArgumentMatchers.anyInt
+import org.mockito.Mock
+import org.mockito.Mockito.`when`
+import org.mockito.junit.MockitoJUnitRunner
+
+@RunWith(MockitoJUnitRunner::class)
+internal class ExitInfoCallbackTest {
+
+    private lateinit var exitInfoCallback: ExitInfoCallback
+
+    @Mock
+    private lateinit var context: Context
+
+    private val eventEnhancer: (Event, ApplicationExitInfo) -> Unit = { _, _ -> }
```

Comment:
Using a mocked `eventEnhancer` would allow the tests to `verify` that it was called when the session IDs match up